cmd/utils/flags.go

```diff
@@ -273,6 +274,58 @@ var (
 		Usage:    "Manually specify the Cancun fork timestamp, overriding the bundled setting",
 		Category: flags.EthCategory,
 	}
+	// Beacon client light sync settings
+	BeaconApiFlag = &cli.StringFlag{
+		Name:     "beacon.api",
+		Usage:    "Beacon node (CL) light client API URL (currently only supports LodeStar)",
+		Category: flags.BeaconCategory,
+	}
+	BeaconApiHeaderFlag = &cli.StringFlag{
+		Name:     "beacon.api.header",
+		Usage:    "Remote beacon node API custom HTTP header fields (\"key:value,key:value\")",
+		Category: flags.BeaconCategory,
```

Comment:
We currently use a StringSliceFlag where we have this in another place -- the nifty thing with that is that it can be given multiple times

```
	HttpHeaderFlag = &cli.StringSliceFlag{
		Name:     "header",
		Aliases:  []string{"H"},
		Usage:    "Pass custom headers to the RPC server when using --" + RemoteDBFlag.Name + " or the geth attach console. This flag can be given multiple times.",
		Category: flags.APICategory,
	}

``` 
can be used like `geth -H "foo=bar" -H "baz=bazonk"`, just like curl headers

Reply:
done